src/WebJobs.Script.SiteExtension/New-PrivateSiteExtension.ps1

```diff
@@ -0,0 +1,130 @@
+<#
+    .SYNOPSIS
+    Produces a private site extension.
+
+    .DESCRIPTION
+    Takes in a published site extension and produces a private site extension.
+
+    .PARAMETER InputPath
+    The path of the published 'SiteExtension'. Leave null to scan for root in a child from here.
+
+    .PARAMETER OutputPath
+    The path to produce the private site extension to. When zipping, this is the name of the zip file. Leave null to compute this name.
+
+    .PARAMETER Bitness
+    The bitness to produce the private site extension with. Default is '32bit'.
+
+    .PARAMETER Zip
+    [Switch] Include to produce site extension as a zip.
+
+    .PARAMETER Force
+    [Switch] Include to overwrite existing files.
+
+    .INPUTS
+    None. You can't pipe objects to Update-Month.ps1.
+
+    .OUTPUTS
+    None. Update-Month.ps1 doesn't generate any output.
+#>
+
+param (
+    [string] $InputPath = $null,
+    [string] $OutputPath = $null,
+    [ValidateSet('x64', '64bit', 'x86', '32bit')][string] $Bitness = '32bit',
```

Comment:
Do private site extensions even work with 64? I think only 32bit works today?

Reply:
No idea! I just added 64bit option here because 'why not'. It defaults to 32bit though

Reply:
![Screenshot 2025-01-22 at 5 36 29 PM](https://github.com/user-attachments/assets/b65f03e9-91a1-4a33-9e8d-941750aea20c)

Actually, I do see 64bit on the platform here, maybe private site extensions do work with this and the documentation for that is just outdated? Only way to double check is to test both 32 and 64 bit pse deployment

Reply:
I validated 32bit. I can also validate 64bit tomorrow.

Reply:
I think publishing a 64-bit version is a good idea. We've encountered several instances where the testing team deployed an app with the default 64-bit platform setting but paired it with a 32-bit extension. This mismatch caused the app to fail, and we had to spend time investigating and pointing out the bitness issue.

Reply:
Checking this morning, it looks like my app deployed and I was able to invoke the 64 bit as well. In summary, I think we should: 

1) keep both options in the script
2) default to 64bit

Reply:
Updated to 64bit default